kuma/wiki/apps.py

```diff
@@ -144,12 +144,14 @@ def on_document_spam_attempt_save(
         if raw or not created:
             # Only send for new instances, not fixtures or edits
             return
+        DocumentSpamAttempt = self.get_model('DocumentSpamAttempt')
         subject = u'[MDN] Wiki spam attempt recorded'
         if instance.document:
             subject = u'%s for document %s' % (subject, instance.document)
         elif instance.title:
             subject = u'%s with title %s' % (subject, instance.title)
-        body = render_to_string('wiki/email/spam.ltxt',
-                                {'spam_attempt': instance})
+        is_error = instance.review == DocumentSpamAttempt.AKISMET_ERROR
+        context = {'spam_attempt': instance, 'is_error': is_error}
```

Comment:
Good point. I was avoiding it because I thought I had to do ``is_error == DocumentSpamAttempt.AKISMET_ERROR``, and didn't want to pass the model. My C++ brain is showing again. Will make the change.